src/ExpressionEvaluator/Directory.Build.props

```diff
@@ -2,5 +2,8 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+
+    <!-- NuGet: Package might not work without long file path support. -->
+    <NoWarn>$(NoWarn);NU5123</NoWarn>
```

Comment:
Why put this here vs. `eng/Settings.props`? Our entire repo requires long path support (we warn on bulid if it's not enabled). So seems likke we should just disable this for everything.

Reply:
This warning is about the customer installing the package, not the build itself. We are not gonna publish EE packages for customers to consume so it doesn't affect anyone but our infra if they have long file names.